### ISSUE TITLE
plot penalized loss in tensorboard

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#422]
ploss= ell(theta)+ mu R(theta),  note that mu is changing, so ploss will also change with iteration

**Comment by drEast** [comment]
solved in #426
